src/student/components/sidebar/Sidebar.tsx

```diff
@@ -40,13 +40,16 @@ const StudentSidebar: React.FC<StudentSidebarProps> = ({ currentView }) => {
       icon: FaHome,
       path: StudentRoutes.Overview,
       color: "#3B82F6",
+      badge: "",
     },
     {
       title: "Mi Billetera",
       icon: FaWallet,
       path: StudentRoutes.Wallet,
       color: "#10B981",
-      badge: currentStudent?.wallet.balance,
+      badge: currentStudent?.wallet.balance
+        ? currentStudent.wallet.balance
+        : "0",
```

Comment:
```suggestion
      badge: currentStudent?.wallet.balance ?? "0"
```

En vez de usar el ? podes usar ?? ya que mostraria 0 solo cuando se null, sino mostraria el balance del student.